lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -76,7 +76,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
    */
   static computeWaste(image, viewportDimensions, networkRecords) {
     const networkRecord = networkRecords.find(record => record.url === image.src);
-    if (!image.resourceSize || !networkRecord) {
+    if (!image.resourceSize || !networkRecord || image.loading === 'lazy' ||
+        image.loading === 'eager') {
```

Comment:
I tried adding `auto` in here as well, but since all images have `loading='auto'` by default, all the `FAIL(offscreen)` images in `smoke byte` were marked as passing. Open to suggestions!